ci/validate-version-bump.sh

```diff
@@ -20,3 +20,4 @@ echo "Base branch  is $base_sha"
 echo "Current head is $head_sha"
 
 cargo bump-check --baseline-rev "$base_sha" --head-rev "$head_sha"
+cargo semver-checks --workspace --baseline-rev "$base_sha"
```

Comment:
Doesn't the `cargo semver-checks` also need the `base` -> `referenced` conversion done as well or else if the current version makes a breaking change but major is already bumped from a prior commit, this will fail when it shouldn't